code/modules/mining/machine_vending.dm

```diff
@@ -318,6 +351,22 @@
 	SSblackbox.record_feedback("tally", "mining_voucher_redeemed", 1, selection)
 	qdel(voucher)
 
+  /*
+   check_menu: Checks if we are allowed to interact with a radial menu
+  
+   Arguments:
+   redeemer The mob interacting with a menu
+   voucher The mining voucher item
+   */
+/obj/machinery/mineral/equipment_vendor/free_miner/check_menu(obj/item/mining_voucher/voucher, mob/living/redeemer)
```

Comment:
You don't need this, if you call check_menu it would just go to the normal equipment_vendor check_menu, which is the exact same code

Reply:
yea i talked with hopek about that and wasn't really sure what to do with it, ill remove it

Reply:
Sorry I misunderstood I thought you were going to add additional functionality to the freeminer one vs the regular one otherwise there is no point to make a separate one.